content/hi/docs/reference/glossary/addons.md

```diff
@@ -0,0 +1,16 @@
+---
+title: ऐड-ऑन
+id: addons
+date: 2019-12-15
+full_link: /docs/concepts/cluster-administration/addons/
+short_description: >
+  संसाधन जो कुबेरनेट्स की कार्यक्षमता का विस्तार करते हैं।
+
+aka:
+tags:
+- tool
+---
+  संसाधन जो कुबेरनेट्स की कार्यक्षमता का विस्तार करते हैं।
+
+<!--more-->
+[ऐड-ऑन इंस्टॉल करना](/docs/concepts/cluster-administration/addons/) अपने क्लस्टर के साथ ऐड-ऑन का उपयोग करने के बारे में अधिक बताता है, और कुछ लोकप्रिय ऐड-ऑन को सूचीबद्ध करता है।
```

Comment:
```suggestion
[ऐड-ऑन इंस्टॉल करना](/docs/concepts/cluster-administration/addons/) अपने क्लस्टर के साथ ऐड-ऑन का उपयोग करने के बारे में अधिक जानकारी देता है, और कुछ लोकप्रिय ऐड-ऑन को सूचीबद्ध करता है।
```
[Nit] Sounds more formal to me

Reply:
Thank @anubha-v-ardhan for the suggestion. I have made the changes.